File: sdk/java/src/main/java/com/hjtpx/captcha/exception/ValidationException.java
package com.hjtpx.captcha.exception;

public class ValidationException extends CaptchaException {
    public ValidationException(String message) {
        super(message, "VALIDATION_ERROR", false);
    }

    public ValidationException(String message, Throwable cause) {
        super(message, "VALIDATION_ERROR", false, cause);
    }
}

File: sdk/csharp/src/Hjtpx.Captcha.Sdk/Exceptions/ValidationException.cs
namespace Hjtpx.Captcha.Exceptions;

public class ValidationException : CaptchaException
{
    public ValidationException(string message)
        : base(message, "VALIDATION_ERROR", false)
    {
    }

    public ValidationException(string message, Exception innerException)
        : base(message, "VALIDATION_ERROR", false, innerException)
    {
    }
}
